doc/building.md

```diff
@@ -1092,16 +1092,33 @@ apt install g++-aarch64-linux-gnu gcc-aarch64-linux-gnu
 
   * Create chroot on the *build* system, configuring it for *target* system:
 ```
-sudo qemu-debootstrap --arch=arm64 --verbose \
-       --include=fakeroot,build-essential,libx11-dev,libxext-dev,libxrender-dev,libxrandr-dev,libxtst-dev,libxt-dev,libcups2-dev,libfontconfig1-dev,libasound2-dev,libfreetype6-dev,libpng12-dev \
-       --resolve-deps jessie /chroots/arm64 http://httpredir.debian.org/debian/
+sudo qemu-debootstrap \
+  --arch=arm64 \
+  --verbose \
+  --include=fakeroot,symlinks,build-essential,libx11-dev,libxext-dev,libxrender-dev,libxrandr-dev,libxtst-dev,libxt-dev,libcups2-dev,libfontconfig1-dev,libasound2-dev,libfreetype6-dev,libpng-dev \
+  --resolve-deps \
+  buster \
+  ~/sysroot-arm64 \
+  http://httpredir.debian.org/debian/
+```
+
+  * Make sure the symlinks inside the newly created chroot point to proper locations:
+
+```
```

Comment:
I think it is important that you do not have an empty line between the bullet line and the code block, to get nicely formatted html. I know markdown is typically fiddly in such cases, and I notice that this is how the other bullets are written.

Reply:
Right! Fixed that.

Reply:
Well, and `make update-build-docs` made no changes after this, so I guess pandoc did not care. Maybe it is missing indents...

Reply:
Yes, indents were incorrect. Fixed, and now generated html looks fine.